cub/test/catch2_test_iterator.cu

```diff
@@ -0,0 +1,277 @@
+/******************************************************************************
+ * Copyright (c) 2011, Duane Merrill.  All rights reserved.
+ * Copyright (c) 2011-2018, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I believe we would need to add our copyright too now

Reply:
Discussed offline. All good.